Presentation.Web/Models/External/V2/Request/UnBoundedPaginationQuery.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Core.ApplicationServices.Shared;
+
+namespace Presentation.Web.Models.External.V2.Request
+{
+    /// <summary>
+    /// Defines the an unbounded pagination query parameter set
+    /// </summary>
+    public class UnboundedPaginationQuery : IStandardPaginationQueryParameters
+
+    {
+        /// <summary>
+        /// 0-based page number. Use this parameter to page through the requested collection.
+        /// Range: [0,2^31] Default: 0
+        /// </summary>
+        [Range(0, int.MaxValue)]
+        public int? Page { get; set; } = null;
```

Comment:
Hvad hvis denne ikke sættes men den anden bliver sat. Så er det vel stadig hele resultatsættet der returneres? 
Så kan kommentaren "If left unspecified, the entire result set will be returned." nok også tilføjes her

Reply:
Tja den er svær for det er rigtigt at de to parametre afhænger af hinanden så forklaringen bør nok være at den skal sættes i kombination med pageSize - ellers vil den ikke få nogen betydning fordi vi ikke har en fallback. Jeg strikker noget sammen